docs/kubernetes/user-guides/windows.md

```diff
@@ -0,0 +1,27 @@
+# Kubernetes CSI Windowos User Guide
```

Comment:
Any other steps needed to use windows? Such as StorageClass settings? Can we add example Pod/pvc/storageclass specs?

Reply:
storage class for linux works just works for linux too.
 I will add example

Reply:
added example